src/Filament/Actions/CloneAction.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace Portable\FilaCms\Filament\Actions;
+
+use Filament\Tables\Actions\Action;
+use Illuminate\Database\Eloquent\Model;
+use Filament\Notifications\Notification;
+
+class CloneAction extends Action
+{
+    protected $currentFile;
+
+    public static function getDefaultName(): ?string
+    {
+        return 'clone';
+    }
+
+    protected function setUp(): void
+    {
+        parent::setUp();
```

Comment:
Nice work!  I think we should redirect them to editing the newly cloned item though